modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1516,6 +1516,18 @@ public ObjectEntry updateObjectEntry(
 			_objectDefinitionPersistence.findByPrimaryKey(
 				objectEntry.getObjectDefinitionId());
 
+		Map<String, Serializable> existingValues = objectEntry.getValues();
+
+		for (Map.Entry<String, Serializable> entry :
+				existingValues.entrySet()) {
+
+			String key = entry.getKey();
+
+			if (!values.containsKey(key)) {
+				values.put(key, existingValues.get(key));
+			}
+		}
+
```

Comment:
@p-albuquerque This code will change the PUT method's expected behavior of removing values absent from the request body.

To avoid blocking the LPP ticket, let's keep your previous approach for now.